webclient/components/def-processors/crud/manage-rows-of-table-in-client-side-orm.js

```diff
@@ -264,6 +264,8 @@ Decision: We will make arOrmRowsCached as a 3D array. Where the 1st D will be en
 
     if (!pFldsForNonEmptyCheck) return
 
+    // $fetch use for Load data from the IndexedDB store associated to a model and persist them in the Vuex Store
+    this.$fetch()
```

Comment:
Data from local storage should only be retrieved during the mounting phase. 

When user is doing r/w 
1. all wrties happned to loacal storage + ORM
2. All reads happen from ORM.

This is same behavior like mariadb api calls